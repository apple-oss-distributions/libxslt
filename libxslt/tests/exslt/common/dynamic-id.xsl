<xsl:stylesheet
    version="1.0"
    xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
    xmlns:exsl="http://exslt.org/common">

<xsl:output indent="yes"/>

<xsl:template name="dynamic-id">
    <id>
        <xsl:value-of select="generate-id(exsl:node-set('string'))"/>
    </id>
</xsl:template>

<xsl:template match="/">
    <result>
        <xsl:call-template name="dynamic-id"/>
        <xsl:call-template name="dynamic-id"/>
        <xsl:call-template name="dynamic-id"/>
        <xsl:call-template name="dynamic-id"/>
        <xsl:call-template name="dynamic-id"/>
        <xsl:call-template name="dynamic-id"/>
        <xsl:call-template name="dynamic-id"/>
        <xsl:call-template name="dynamic-id"/>
        <xsl:call-template name="dynamic-id"/>
        <xsl:call-template name="dynamic-id"/>
    </result>
</xsl:template>

</xsl:stylesheet>
