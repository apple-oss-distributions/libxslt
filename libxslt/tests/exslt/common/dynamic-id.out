<?xml version="1.0"?>
<result xmlns:exsl="http://exslt.org/common">
  <id>id1</id>
  <id>id2</id>
  <id>id3</id>
  <id>id4</id>
  <id>id5</id>
  <id>id6</id>
  <id>id7</id>
  <id>id8</id>
  <id>id9</id>
  <id>id10</id>
</result>
